installation on atom failed. OS: win8 64bit

Installing “markdown-preview-plus@1.5.0” failed.Hide output…

```
> runas@2.0.0 install C:\Users\Tal\AppData\Local\Temp\apm-install-dir-115514-2964-mqnnf4\node_modules\markdown-preview-plus\node_modules\pathwatcher\node_modules\runas
> node-gyp rebuild


C:\Users\Tal\AppData\Local\Temp\apm-install-dir-115514-2964-mqnnf4\node_modules\markdown-preview-plus\node_modules\pathwatcher\node_modules\runas>node "C:\Users\Tal\AppData\Local\atom\app-0.209.0\resources\app\apm\node_modules\npm\bin\node-gyp-bin\\..\..\node_modules\node-gyp\bin\node-gyp.js" rebuild 

> oniguruma@4.2.2 install C:\Users\Tal\AppData\Local\Temp\apm-install-dir-115514-2964-mqnnf4\node_modules\markdown-preview-plus\node_modules\highlights\node_modules\first-mate\node_modules\oniguruma
> node-gyp rebuild


C:\Users\Tal\AppData\Local\Temp\apm-install-dir-115514-2964-mqnnf4\node_modules\markdown-preview-plus\node_modules\highlights\node_modules\first-mate\node_modules\oniguruma>node "C:\Users\Tal\AppData\Local\atom\app-0.209.0\resources\app\apm\node_modules\npm\bin\node-gyp-bin\\..\..\node_modules\node-gyp\bin\node-gyp.js" rebuild 

npm WARN deprecated CSSselect@0.4.1: the module is now available as 'css-select'
npm WARN deprecated CSSwhat@0.4.7: the module is now available as 'css-what'
gypgypnpm ERR! Windows_NT 6.2.9200
npm ERR! argv "C:\\Users\\Tal\\AppData\\Local\\atom\\app-0.209.0\\resources\\app\\apm\\bin\\\\node.exe" "C:\\Users\\Tal\\AppData\\Local\\atom\\app-0.209.0\\resources\\app\\apm\\node_modules\\npm\\bin\\npm-cli.js" "--globalconfig" "C:\\Users\\Tal\\.atom\\.apm\\.apmrc" "--userconfig" "C:\\Users\\Tal\\.atom\\.apmrc" "install" "C:\\Users\\Tal\\AppData\\Local\\Temp\\d-115514-2964-1l06avu\\package.tgz" "--target=0.22.0" "--arch=ia32"
npm ERR! node v0.10.35
npm ERR! npm  v2.5.1
npm ERR! code ELIFECYCLE

npm ERR! runas@2.0.0 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the runas@2.0.0 install script 'node-gyp rebuild'.
npm ERR! This is most likely a problem with the runas package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get their info via:
npm ERR!     npm owner ls runas
npm ERR! There is likely additional logging output above.
```

@Galadirith I guess I'm experiencing pretty much the same issue on Win 7 / 64-bit. Here's what i get

```
C:\Users\q6q>apm install markdown-preview-plus
Installing markdown-preview-plus to C:\Users\q6q\.atom\packages failed

> runas@2.0.0 install C:\Users\q6q\AppData\Local\Temp\apm-install-dir-115613-13128-9fe2u\node_modules\markdown-preview-plus\node_modules\pathwatcher\node_modules\runas
> node-gyp rebuild


C:\Users\q6q\AppData\Local\Temp\apm-install-dir-115613-13128-9fe2u\node_modules\markdown-preview-plus\node_modules\pathwatcher\node_modules\runas>node "C:\Users\q6q\AppData\Local\atom\app-1.0.2\resour
ces\app\apm\node_modules\npm\bin\node-gyp-bin\\..\..\node_modules\node-gyp\bin\node-gyp.js" rebuild

> oniguruma@4.2.2 install C:\Users\q6q\AppData\Local\Temp\apm-install-dir-115613-13128-9fe2u\node_modules\markdown-preview-plus\node_modules\highlights\node_modules\first-mate\node_modules\oniguruma
> node-gyp rebuild


C:\Users\q6q\AppData\Local\Temp\apm-install-dir-115613-13128-9fe2u\node_modules\markdown-preview-plus\node_modules\highlights\node_modules\first-mate\node_modules\oniguruma>node "C:\Users\q6q\AppData\
Local\atom\app-1.0.2\resources\app\apm\node_modules\npm\bin\node-gyp-bin\\..\..\node_modules\node-gyp\bin\node-gyp.js" rebuild

npm WARN deprecated CSSselect@0.4.1: the module is now available as 'css-select'
npm WARN deprecated CSSwhat@0.4.7: the module is now available as 'css-what'
gypgypnpm ERR! Windows_NT 6.1.7601
npm ERR! argv "C:\\Users\\q6q\\AppData\\Local\\atom\\app-1.0.2\\resources\\app\\apm\\bin\\\\node.exe" "C:\\Users\\q6q\\AppData\\Local\\atom\\app-1.0.2\\resources\\app\\apm\\node_modules\\npm\\bin\\npm
-cli.js" "--globalconfig" "C:\\Users\\q6q\\.atom\\.apm\\.apmrc" "--userconfig" "C:\\Users\\q6q\\.atom\\.apmrc" "install" "C:\\Users\\q6q\\AppData\\Local\\Temp\\d-115613-13128-5rpy0d\\package.tgz" "--t
arget=0.22.0" "--arch=ia32" "--msvs_version=2013"
npm ERR! node v0.10.35
npm ERR! npm  v2.5.1
npm ERR! code ELIFECYCLE

npm ERR! runas@2.0.0 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the runas@2.0.0 install script 'node-gyp rebuild'.
npm ERR! This is most likely a problem with the runas package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get their info via:
npm ERR!     npm owner ls runas
npm ERR! There is likely additional logging output above.
```

Here's what i get from apm
```
C:\Users\q6q>apm -v
apm  1.0.1
npm  2.5.1
node 0.10.35
python
git 1.9.5.msysgit.1
visual studio 2013
```
```
C:\Users\q6q>apm install --check
Checking for native build tools done
```

Any ideas?

I seem to have the same/similar problem. I tried using git bash, but that didn't seem to solve the problem.

I get the following error log both using the inbuilt installer and from git bash command line:

```
> runas@2.0.0 install C:\Users\joolarjo\AppData\Local\Temp\apm-install-dir-11574-1432-jzh595\node_modules\markdown-preview-plus\node_modules\pathwatcher\node_modules\runas
> node-gyp rebuild

C:\Users\joolarjo\AppData\Local\Temp\apm-install-dir-11574-1432-jzh595\node_modules\markdown-preview-plus\node_modules\pathwatcher\node_modules\runas>node "C:\MyTemp\Atom\resources\app\apm\node_modules\npm\bin\node-gyp-bin\\..\..\node_modules\node-gyp\bin\node-gyp.js" rebuild

> oniguruma@4.2.2 install C:\Users\joolarjo\AppData\Local\Temp\apm-install-dir-11574-1432-jzh595\node_modules\markdown-preview-plus\node_modules\highlights\node_modules\first-mate\node_modules\oniguruma
> node-gyp rebuild

C:\Users\joolarjo\AppData\Local\Temp\apm-install-dir-11574-1432-jzh595\node_modules\markdown-preview-plus\node_modules\highlights\node_modules\first-mate\node_modules\oniguruma>node "C:\MyTemp\Atom\resources\app\apm\node_modules\npm\bin\node-gyp-bin\\..\..\node_modules\node-gyp\bin\node-gyp.js" rebuild

npm WARN deprecated CSSselect@0.4.1: the module is now available as 'css-select'

npm WARN deprecated CSSwhat@0.4.7: the module is now available as 'css-what'
gypgypnpm ERR! Windows_NT 6.1.7601
npm ERR! argv "c:\\MyTemp\\Atom\\resources\\app\\apm\\bin\\node.exe" "C:\\MyTemp\\Atom\\resources\\app\\apm\\node_modules\\npm\\bin\\npm-cli.js" "--globalconfig" "C:\\Users\\joolarjo\\.atom\\.apm\\.apmrc" "--userconfig" "C:\\Users\\joolarjo\\.atom\\.apmrc" "install" "C:\\Users\\joolarjo\\AppData\\Local\\Temp\\d-11574-1432-olzuhv\\package.tgz" "--target=0.22.0" "--arch=ia32"
npm ERR! node v0.10.35
npm ERR! npm  v2.5.1
npm ERR! code ELIFECYCLE

npm ERR! runas@2.0.0 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the runas@2.0.0 install script 'node-gyp rebuild'.
npm ERR! This is most likely a problem with the runas package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get their info via:
npm ERR!     npm owner ls runas
npm ERR! There is likely additional logging output above.
```

I got this working with Linux -- great work. It is the missing link in a great markdown-based workflow.

All of these are actually `apm.cmd`, but I noticed no difference with Git Bash. Here's the output:

`apm -v`

```
apm  1.0.1
npm  2.5.1
node 0.10.35
python
git 1.7.11.msysgit.1
visual studio
```

`apm  --check`

```
apm - Atom Package Manager powered by https://atom.io

Usage: apm <command>

where <command> is one of:
    clean, config, dedupe, deinstall, delete, dev, develop, docs, erase,
    featured, home, init, install, link, linked, links, list, ln, lns, login,
    ls, open, outdated, publish, rebuild, rebuild-module-cache, remove, rm,
    search, show, star, starred, stars, test, uninstall, unlink, unpublish,
    unstar, update, upgrade, view.

Run `apm help <command>` to see the more details about a specific command.

Options:
  --color        Enable colored output                                     [boolean] [default: true]
  -v, --version  Print the apm version
  -h, --help     Print this usage message

  Prefix an option with `no-` to set it to false such as --no-color to disable
  colored output.
```

All I get to a .log-file is the following, even though I can see some errors running past me in the console window.

```
Installing markdown-preview-plus to C:\Users\joolarjo\.atom\packages 0.22.0

> runas@2.0.0 install C:\Users\joolarjo\AppData\Local\Temp\apm-install-dir-11575-7372-11nee90\node_modules\markdown-preview-plus\node_modules\pathwatcher\node_modules\runas
> node-gyp rebuild


CLINK C:\Users\joolarjo\AppData\Local\Temp\apm-install-dir-11575-7372-11nee90\node_modules\markdown-preview-plus\node_modules\pathwatcher\node_modules\runas>node "C:\MyTemp\Atom\resources\app\apm\node_modules\npm\bin\node-gyp-bin\\..\..\node_modules\node-gyp\bin\node-gyp.js" rebuild 

> oniguruma@4.2.2 install C:\Users\joolarjo\AppData\Local\Temp\apm-install-dir-11575-7372-11nee90\node_modules\markdown-preview-plus\node_modules\highlights\node_modules\first-mate\node_modules\oniguruma
> node-gyp rebuild


CLINK C:\Users\joolarjo\AppData\Local\Temp\apm-install-dir-11575-7372-11nee90\node_modules\markdown-preview-plus\node_modules\highlights\node_modules\first-mate\node_modules\oniguruma>node "C:\MyTemp\Atom\resources\app\apm\node_modules\npm\bin\node-gyp-bin\\..\..\node_modules\node-gyp\bin\node-gyp.js" rebuild 
[31mfailed
[39m
```

If I take all of the console scrollback, I can salvage this much:

```
  localAddress: undefined,
  headers:
   { 'accept-encoding': 'gzip',
     version: '2.5.1',
     accept: 'application/json',
     referer: 'install [REDACTED]',
     'npm-session': '484bed288478e609',
     'user-agent': 'npm/2.5.1 node/v0.10.35 win32 ia32',
     'if-none-match': '"EZ8MXN3H4NCB2NOG0SQN5HF4F"' },
  method: 'GET',
  url:
   { protocol: 'https:',
     slashes: true,
     auth: null,
     host: 'registry.npmjs.org',
     port: null,
     hostname: 'registry.npmjs.org',
     hash: null,
     search: null,
     query: null,
     pathname: '/argparse',
     path: '/argparse',
     href: 'https://registry.npmjs.org/argparse' },
  callback: [Function],
  uri: undefined }
REQUEST make request https://registry.npmjs.org/argparse
REQUEST { encoding: null,
  followRedirect: true,
  agent:
   { domain: null,
     _events: { free: [Function] },
     _maxListeners: 10,
     options:
      { keepAlive: true,
        localAddress: undefined,
        rejectUnauthorized: true,
        ca: null,
        cert: undefined,
        key: null },
     requests: {},
     sockets: { 'registry.npmjs.org:443': [Object] },
     maxSockets: 5,
     createConnection: [Function: createConnection] },
  ca: null,
  key: null,
  cert: null,
  strictSSL: true,
  localAddress: undefined,
  headers:
   { 'accept-encoding': 'gzip',
     version: '2.5.1',
     accept: 'application/json',
     referer: 'install [REDACTED]',
     'npm-session': '484bed288478e609',
     'user-agent': 'npm/2.5.1 node/v0.10.35 win32 ia32',
     'if-none-match': '"DMNY25706PTYN7DXW58UUFEBF"' },
  method: 'GET',
  url:
   { protocol: 'https:',
     slashes: true,
     auth: null,
     host: 'registry.npmjs.org',
     port: null,
     hostname: 'registry.npmjs.org',
     hash: null,
     search: null,
     query: null,
     pathname: '/esprima',
     path: '/esprima',
     href: 'https://registry.npmjs.org/esprima' },
  callback: [Function],
  uri: undefined }
REQUEST make request https://registry.npmjs.org/esprima
REQUEST onRequestResponse https://registry.npmjs.org/argparse 304 { date: 'Wed,
05 Aug 2015 08:24:48 GMT',
  via: '1.1 varnish',
  'cache-control': 'max-age=60',
  etag: '"EZ8MXN3H4NCB2NOG0SQN5HF4F"',
  age: '18',
  connection: 'keep-alive',
  'x-served-by': 'cache-fra1230-FRA',
  'x-cache': 'HIT',
  'x-cache-hits': '3',
  'x-timer': 'S1438763088.433183,VS0,VE0',
  vary: 'Accept' }
REQUEST finish init function https://registry.npmjs.org/argparse
REQUEST response end https://registry.npmjs.org/argparse 304 { date: 'Wed, 05 Au
g 2015 08:24:48 GMT',
  via: '1.1 varnish',
  'cache-control': 'max-age=60',
  etag: '"EZ8MXN3H4NCB2NOG0SQN5HF4F"',
  age: '18',
  connection: 'keep-alive',
  'x-served-by': 'cache-fra1230-FRA',
  'x-cache': 'HIT',
  'x-cache-hits': '3',
  'x-timer': 'S1438763088.433183,VS0,VE0',
  vary: 'Accept' }
REQUEST end event https://registry.npmjs.org/argparse
REQUEST emitting complete https://registry.npmjs.org/argparse
REQUEST onRequestResponse https://registry.npmjs.org/esprima 304 { date: 'Wed, 0
5 Aug 2015 08:24:48 GMT',
  via: '1.1 varnish',
  'cache-control': 'max-age=60',
  etag: '"DMNY25706PTYN7DXW58UUFEBF"',
  age: '39',
  connection: 'keep-alive',
  'x-served-by': 'cache-fra1243-FRA',
  'x-cache': 'HIT',
  'x-cache-hits': '5',
  'x-timer': 'S1438763088.450166,VS0,VE0',
  vary: 'Accept' }
REQUEST finish init function https://registry.npmjs.org/esprima
REQUEST response end https://registry.npmjs.org/esprima 304 { date: 'Wed, 05 Aug
 2015 08:24:48 GMT',
  via: '1.1 varnish',
  'cache-control': 'max-age=60',
  etag: '"DMNY25706PTYN7DXW58UUFEBF"',
  age: '39',
  connection: 'keep-alive',
  'x-served-by': 'cache-fra1243-FRA',
  'x-cache': 'HIT',
  'x-cache-hits': '5',
  'x-timer': 'S1438763088.450166,VS0,VE0',
  vary: 'Accept' }
REQUEST end event https://registry.npmjs.org/esprima
REQUEST emitting complete https://registry.npmjs.org/esprima
REQUEST { encoding: null,
  followRedirect: true,
  agent:
   { domain: null,
     _events: { free: [Function] },
     _maxListeners: 10,
     options:
      { keepAlive: true,
        localAddress: undefined,
        rejectUnauthorized: true,
        ca: null,
        cert: undefined,
        key: null },
     requests: {},
     sockets: {},
     maxSockets: 5,
     createConnection: [Function: createConnection] },
  ca: null,
  key: null,
  cert: null,
  strictSSL: true,
  localAddress: undefined,
  headers:
   { 'accept-encoding': 'gzip',
     version: '2.5.1',
     accept: 'application/json',
     referer: 'install [REDACTED]',
     'npm-session': '484bed288478e609',
     'user-agent': 'npm/2.5.1 node/v0.10.35 win32 ia32',
     'if-none-match': '"CVUBWYO6DNN77U6X6NT1N1M2A"' },
  method: 'GET',
  url:
   { protocol: 'https:',
     slashes: true,
     auth: null,
     host: 'registry.npmjs.org',
     port: null,
     hostname: 'registry.npmjs.org',
     hash: null,
     search: null,
     query: null,
     pathname: '/underscore',
     path: '/underscore',
     href: 'https://registry.npmjs.org/underscore' },
  callback: [Function],
  uri: undefined }
REQUEST make request https://registry.npmjs.org/underscore
REQUEST { encoding: null,
  followRedirect: true,
  agent:
   { domain: null,
     _events: { free: [Function] },
     _maxListeners: 10,
     options:
      { keepAlive: true,
        localAddress: undefined,
        rejectUnauthorized: true,
        ca: null,
        cert: undefined,
        key: null },
     requests: {},
     sockets: { 'registry.npmjs.org:443': [Object] },
     maxSockets: 5,
     createConnection: [Function: createConnection] },
  ca: null,
  key: null,
  cert: null,
  strictSSL: true,
  localAddress: undefined,
  headers:
   { 'accept-encoding': 'gzip',
     version: '2.5.1',
     accept: 'application/json',
     referer: 'install [REDACTED]',
     'npm-session': '484bed288478e609',
     'user-agent': 'npm/2.5.1 node/v0.10.35 win32 ia32',
     'if-none-match': '"42XVVLS85OA5X9B930MT3LD6Y"' },
  method: 'GET',
  url:
   { protocol: 'https:',
     slashes: true,
     auth: null,
     host: 'registry.npmjs.org',
     port: null,
     hostname: 'registry.npmjs.org',
     hash: null,
     search: null,
     query: null,
     pathname: '/underscore.string',
     path: '/underscore.string',
     href: 'https://registry.npmjs.org/underscore.string' },
  callback: [Function],
  uri: undefined }
REQUEST make request https://registry.npmjs.org/underscore.string
REQUEST onRequestResponse https://registry.npmjs.org/underscore 304 { date: 'Wed
, 05 Aug 2015 08:24:48 GMT',
  via: '1.1 varnish',
  'cache-control': 'max-age=60',
  etag: '"CVUBWYO6DNN77U6X6NT1N1M2A"',
  age: '52',
  connection: 'keep-alive',
  'x-served-by': 'cache-fra1238-FRA',
  'x-cache': 'HIT',
  'x-cache-hits': '5',
  'x-timer': 'S1438763088.853313,VS0,VE0',
  vary: 'Accept' }
REQUEST finish init function https://registry.npmjs.org/underscore
REQUEST response end https://registry.npmjs.org/underscore 304 { date: 'Wed, 05
Aug 2015 08:24:48 GMT',
  via: '1.1 varnish',
  'cache-control': 'max-age=60',
  etag: '"CVUBWYO6DNN77U6X6NT1N1M2A"',
  age: '52',
  connection: 'keep-alive',
  'x-served-by': 'cache-fra1238-FRA',
  'x-cache': 'HIT',
  'x-cache-hits': '5',
  'x-timer': 'S1438763088.853313,VS0,VE0',
  vary: 'Accept' }
REQUEST end event https://registry.npmjs.org/underscore
REQUEST emitting complete https://registry.npmjs.org/underscore
REQUEST onRequestResponse https://registry.npmjs.org/underscore.string 304 { dat
e: 'Wed, 05 Aug 2015 08:24:48 GMT',
  via: '1.1 varnish',
  'cache-control': 'max-age=60',
  etag: '"42XVVLS85OA5X9B930MT3LD6Y"',
  age: '15',
  connection: 'keep-alive',
  'x-served-by': 'cache-fra1233-FRA',
  'x-cache': 'HIT',
  'x-cache-hits': '1',
  'x-timer': 'S1438763088.870929,VS0,VE0',
  vary: 'Accept' }
REQUEST finish init function https://registry.npmjs.org/underscore.string
REQUEST response end https://registry.npmjs.org/underscore.string 304 { date: 'W
ed, 05 Aug 2015 08:24:48 GMT',
  via: '1.1 varnish',
  'cache-control': 'max-age=60',
  etag: '"42XVVLS85OA5X9B930MT3LD6Y"',
  age: '15',
  connection: 'keep-alive',
  'x-served-by': 'cache-fra1233-FRA',
  'x-cache': 'HIT',
  'x-cache-hits': '1',
  'x-timer': 'S1438763088.870929,VS0,VE0',
  vary: 'Accept' }
REQUEST end event https://registry.npmjs.org/underscore.string
REQUEST emitting complete https://registry.npmjs.org/underscore.string
npm ERR! Windows_NT 6.1.7601
npm ERR! argv "C:\\MyTemp\\Atom\\resources\\app\\apm\\bin\\\\node.exe" "C:\\MyTe
mp\\Atom\\resources\\app\\apm\\node_modules\\npm\\bin\\npm-cli.js" "--globalconf
ig" "C:\\Users\\joolarjo\\.atom\\.apm\\.apmrc" "--userconfig" "C:\\Users\\joolar
jo\\.atom\\.apmrc" "install" "C:\\Users\\joolarjo\\AppData\\Local\\Temp\\d-11575
-7372-1ya082h\\package.tgz" "--target=0.22.0" "--arch=ia32"
npm ERR! node v0.10.35
npm ERR! npm  v2.5.1
npm ERR! code ELIFECYCLE

npm ERR! runas@2.0.0 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the runas@2.0.0 install script 'node-gyp rebuild'.
npm ERR! This is most likely a problem with the runas package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get their info via:
npm ERR!     npm owner ls runas
npm ERR! There is likely additional logging output above.
```

--- a/lib/markdown-preview-view.coffee
+++ b/lib/markdown-preview-view.coffee
@@ -5,7 +5,7 @@ path = require 'path'
 Grim = require 'grim'
 _ = require 'underscore-plus'
 fs = require 'fs-plus'
-{File} = require 'pathwatcher'
+{File} = require 'pathwatcher-without-runas'
 
 renderer = require './renderer'
 UpdatePreview = require './update-preview'

--- a/lib/renderer.coffee
+++ b/lib/renderer.coffee
@@ -8,7 +8,7 @@ roaster = null # Defer until used
 pandocHelper = null # Defer until used
 {scopeForFenceName} = require './extension-helper'
 mathjaxHelper = require './mathjax-helper'
-pathWatcher = require 'pathwatcher'
+pathWatcher = require 'pathwatcher-without-runas'
 
 MarkdownPreviewView = null # Defer until used
 isMarkdownPreviewView = (object) ->

--- a/package.json
+++ b/package.json
@@ -21,7 +21,7 @@
     "fs-plus": "^2.0.0",
     "grim": "^1.2.1",
     "highlights": "^1.0.0",
-    "pathwatcher": "^4.4.3",
+    "pathwatcher-without-runas": "~4.5.0",
     "pdc": "~0.2.2",
     "roaster": "https://github.com/Galadirith/roaster/tarball/marked-mathjax",
     "temp": "^0.8.1",

--- a/spec/markdown-preview-view-spec.coffee
+++ b/spec/markdown-preview-view-spec.coffee
@@ -2,7 +2,6 @@ path = require 'path'
 fs = require 'fs-plus'
 temp = require 'temp'
 MarkdownPreviewView = require '../lib/markdown-preview-view'
-pathWatcher = require 'pathwatcher'
 url = require 'url'
 queryString = require 'querystring'
 

